src/pages/Final.less

```diff
@@ -0,0 +1,15 @@
+.final-page {
+    h1,
+    p {
+        text-align: center;
+    }
+
+    p {
+        margin-bottom: 20px;
+    }
+
+    &__table {
+        max-width: 600px;
```

Comment:
We should use predefined sizes as defined by Bootstrap grid system, shouldn't we?

Reply:
I don't think so.